.github/actions/install_dbt_requirements/action.yaml

```diff
@@ -8,16 +8,25 @@ inputs:
 runs:
   using: composite
   steps:
-      - name: Setup python
-        uses: actions/setup-python@v4
+      - name: Install uv
+        uses: astral-sh/setup-uv@v3
         with:
-          python-version: '3.11'
-          cache: pip
+          enable-cache: true
+          cache-dependency-glob: |
+            ${{ inputs.dbt_project_dir }}/pyproject.toml
+            ${{ inputs.dbt_project_dir }}/uv.lock
```

Comment:
Unlike before, we have to explicitly specify the cache keys for pip dependencies. This is actually probably a good thing insofar as it lets us isolate each of the subproject dependencies in their own cache.